Week 2/Contest (Mar 18)/Okvir/dominic.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+int main() {
+  int m, n;
+  cin >> m >> n;
+  vector<int> d(4);
+  for (auto &it : d) cin >> it;
+  vector<string> words(m);
+  for (auto &it : words) cin >> it;
+
+  int sizeM = m + d[0] + d[3];
+  int sizeN = n + d[1] + d[2];
+
+  string symbols = "#.";
+  vector<vector<char>> grid(sizeM, vector<char>(sizeN));
+  for (int i = 0; i < sizeM; i++) {
+    int l = i%2;
+    for (int j = 0; j < sizeN; j++, l++) {
+      grid[i][j] = symbols[l%2];
+    }
+  }
+
+  for (int i = d[0], l = 0; l < m; i++, l++) {
+    string word = words[l];
```

Comment:
```suggestion
    string& word = words[l];
```
we don't need to copy here